extensions/photoroom-image-editing/src/index.tsx

```diff
@@ -0,0 +1,393 @@
+import {
+  Form,
+  ActionPanel,
+  Action,
+  showToast,
+  Toast,
+  showInFinder,
+  captureException,
+  getPreferenceValues,
+  openExtensionPreferences,
+  confirmAlert,
+  LaunchProps,
+} from "@raycast/api";
+import { useForm, FormValidation } from "@raycast/utils";
+import { useState } from "react";
+import fs from "fs";
+import { pipeline as streamPipeline } from "node:stream/promises";
+import { got, RequestError } from "got";
+import isImage from "is-image";
+import { FormData as FormDataNode } from "formdata-node";
+import { fileFromPath } from "formdata-node/file-from-path";
+import { homedir } from "os";
+import { join } from "path";
+
+type Values = {
+  imagePaths: string[];
+  removeBackground: boolean;
+  backgroundColor: string;
+  backgroundPrompt: string;
+  shadowMode: string;
+  scalingMode: string;
+  snappedToCroppedEdges: boolean;
+  outputSize: string;
+  widthxHeight: string;
+  maxWidth: string;
+  maxHeight: string;
+};
+
+interface Preferences {
+  photoroomApiKey: string;
+}
+
```

Comment:
Is this file duplicated or is just my impression?

Reply:
Yes, you are right, not sure how that happened. I've removed it. 🙏